docs/source/scancode-toolkit/rst_snippets/stdout.rst

```diff
@@ -0,0 +1,9 @@
+Print to ``stdout`` (Terminal)
+------------------------------
+
+If you want to format the output in JSON and print it at stdout, you can replace the JSON filename
+with a "-", like ``--json-pp -`` instead of ``--json-pp output.json``.
+
+The following command will output the scan results in JSON format to ``stdout`` (In the Terminal)::
+
+    ./scancode -clpieu --jsonlines - samples/
```

Comment:
We may want to think about removing `--jsonlines` here and just use `--json-pp -`. We mainly just want to show users how to output to stdout. I just think using different output format in this example complicates this for a new user more than it should. 

Eventually we can have another page detailing the other various output formats.

Reply:
Yeah sure!